### PR TITLE
add mlflow code samples

### PR DESCRIPTION
**Pull request by parano**

### CODE REVIEW

**Review by ssheng — APPROVED** [review]

File: mlflow/pytorch/mnist.py [hunk 1]
```diff
@@ -223,7 +223,7 @@ def log_scalar(name, value, step):
         # Option 2: Import logged mlflow model to BentoML for serving:
         model_uri = mlflow.get_artifact_uri("pytorch-model")
         bento_model_2 = bentoml.mlflow.import_model(
-            'mlflow-pytorch-mnist',
```
Comment (aarnphm) [inline mlflow/pytorch/mnist.py]: we should at least do string normalization here right @sauyon ?

**Review by aarnphm — APPROVED** [review]

**Comment by aarnphm** [comment]
oh I approve by mistake, I need to test some of this out first.

File: mlflow/pytorch/service.py [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+import torch
+
+mnist_runner = bentoml.mlflow.get('mlflow_pytorch_mnist:latest').to_runner()
+
+svc = bentoml.Service('mlflow_pytorch_mnist_demo', runners=[ mnist_runner ])
+
+input_spec = bentoml.io.NumpyNdarray(
+    dtype="float32",
+    shape=[-1, 1, 28, 28],
+    enforce_shape=True,
```
Comment (parano) [inline mlflow/pytorch/service.py:13]:
```suggestion
```

**Review by aarnphm — APPROVED** [review]
I test out torchscript locally, seems to work fine. I believe in you on other projects.
